specification/apicenter/ApiCenter.DataApi/models/api.tsp

```diff
@@ -19,32 +32,34 @@ model Api {
   description?: string;
 
   @doc("Kind of API. For example, REST or GraphQL.")
-  kind: string;
+  kind: ApiKind;
 
   @doc("Current lifecycle stage of the API.")
-  lifecycleStage?: ApiLifecycleStage;
+  @visibility("read")
+  lifecycleStage?: LifecycleStage;
 
   @doc("Terms of service for the API.")
-  termsOfService?: ApiTermsOfService;
+  termsOfService?: TermsOfService;
 
   @doc("The license information for the API.")
-  license?: ApiLicenseModel;
+  license?: License;
 
   @doc("External documentation")
-  externalDocumentation?: Array<ApiExternalDocumentation>;
+  externalDocumentation?: ExternalDocumentation[];
 
   @doc("Points of contact for the API.")
-  contacts?: Array<ApiContact>;
+  contacts?: Array<Contact>;
```

Comment:
Nit: You use the `T[]` array syntax in some places but not others. I recommend you be consistent throughout.